cmd/release-controller/sync.go

```diff
@@ -276,6 +276,15 @@ func countUnreadyReleases(release *Release, tags []*imagev1.TagReference) int {
 	return unreadyTagCount
 }
 
+func isReleaseFailing(tags []*imagev1.TagReference) bool {
+	for i := 0; i < 5 && i < len(tags); i++ {
```

Comment:
This needs to be the per job constant for retained failures

Reply:
where are those constants defined?

Reply:
https://github.com/openshift/release/blob/master/core-services/release-controller/_releases/release-ocp-4.1.json#L7  ?